MyToolsYourToolsClient/src/app/admin-panel/notifications/notifications.component.ts

```diff
@@ -10,7 +10,7 @@ import { NotificationService } from '../../services/notification.service';
 })
 export class NotificationsComponent implements OnInit {
 
-  currentUserId = '1';
+  currentUserId = '87e48e98-e923-49be-a134-caff0c7fae0a';
```

Comment:
to już można zrobić normalnie że w ngOnInit bierzemy jako:
this.currentUserId = localStorage.getItem('auth_key');